src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -36,13 +37,21 @@
                       12:01 AM (Pacific Time). No other filings are allowed until then.</span>
                   </v-tooltip>
                 </v-scale-transition>
+
+                <template v-else-if="isPaid(filing.status)">
```

Comment:
Does this state cover Future effective filings as well. Is the plan for them to return to the filings history list as 'Paid'?

Reply:
It does not cover FE filings as those, I think, will have a new status and different text (UI TBD). At the moment those would fall under the 'else' case and appear as Filed and Paid. Anyway, FE filings are future.

Reply:
I dont think there will be a new status for this from BE and future effective filings must be working right now with Cameron's UI changes unless there is some bug in any of the components involved. There is a ticket to tackle this paid status anyways

Reply:
I think THIS is the ticket to handle the PAID status.

I think we will need a new status to distinguish between paid-but-filer-is-slow-or-broken and paid-and-awaiting-future-effective-date-and-time...? Do we need to discuss/design this?

Reply:
This ticket specifically handles the PAID status (where the EF has processed this filing yet) -- not future effective.

FE filings will be handled in the future.